source/qx/io/jsonrpc/__init__.js

```diff
@@ -1,116 +0,0 @@
-/**
- * This namespace provides a JSON Remote Procedure Call (JSON-RPC) version 2 API .
- * See https://www.jsonrpc.org/specification
- *
- * JSON-RPC v2 is transport-agnostic. We provide an API interface
- * and an HTTP transport implementatio. Other transports
- * based on websockets or other mechanisms can be added later.
- *
- * There are two ways of working with this API.
- *
- * 1) Using the transport objects
- *
- * You can either work with the transports directly. For example, you can send
- * out requests, notifications, and batches like so:
- *
- * <pre>
- *   (async()=>{
- *    const transport = new qx.io.jsonrpc.transport.Http("https://domain.com/endpoint");
- *    transport.getRequest().setAuthentication(new qx.io.request.authentication.Bearer("TOKEN"));
```

Comment:
I think `getRequest` is a transport-specific call, right? It only applies to HTTP? If so, the term "request" is a bit too ambiguous since we also have `qx.io.jsonrpc.message.Request`. Suggest something like `getTransportImpl`.

Reply:
Done.